Created File_Module_Page file and added locators

DariaBoldina opened the pull request:
Milan, please check my code

Commit:
Created File_Module_Page file and added locators

milan-82 reviewed (approved):
Daria eveything looks fine for me

Commit:
Merge branch 'master' into B29G43-122_Files_Daria

Commit:
Updated File_Module_Page file and added some locators
Added data to configuration.properties
Created File_Module_StepDefinitions and added my code
Created FileModule.feature file and wrote scenario
Imported 1_B29G43-139.feature from Jira and run this scenario with CukesRunner file

DariaBoldina commented:
Thank you, Milan. 
Also i pushed my all code. Can you check , please?
